### PR TITLE
Fixed ajbdev/authorizenet-php-api deprecation

### PR DESCRIPTION
**Pull request by herodrigues**
Fixed the authorizenet deprecation and update to its latest version.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
         "symfony/http-foundation": "~2.3",
         "symfony/form": "~2.3",
         "symfony/validator": "~2.6",
-        "ajbdev/authorizenet-php-api": "1.1.*",
+        "authorizenet/authorizenet": "1.8.*",
```
Comment (makasim) [inline composer.json]: It would be good to set it like `~1.8`

Comment (makasim) [inline composer.json]: You have to update this line https://github.com/Payum/Payum/pull/417/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L93 too, and the composer.json of Payum/AuthorizeNet/Aim component

**Comment by makasim** [comment]
@herodrigues Thanks for the PR, does it work for  you?

**Comment by herodrigues** [comment]
Ok, I'm gonna do that

**Comment by herodrigues** [comment]
It's done :+1:

**Comment by makasim** [comment]
@herodrigues merged, thanks for PR

**Pull request merged**